### DISCUSSION TITLE
Add stats to recentGames or add a stats endpoint

### DISCUSSION THREAD

**Discussion by unlimitedcoder2** [HypixelDev/PublicAPI#362] [Ideas]
Occasionally after a game of bedwars a link is sent in chat to see the stats of the game that was just played [example](https://stats.hypixel.net/?id=5ff5ac9f44f402222b1d4b68)

![javaw_Bhc4s8DXBI](https://user-images.githubusercontent.com/16884313/103768666-d520cb00-501a-11eb-8641-c9412138658e.png)
![brave_416ATS0qDa](https://user-images.githubusercontent.com/16884313/103769172-b1aa5000-501b-11eb-83d4-a71aa86e8cbd.png)
I think it would be a good addition to the recentGames endpoint as the data it has is useful

**Comment by mdashlw** [comment]
See https://github.com/HypixelDev/PublicAPI/issues/258#issuecomment-643211487
